apps/src/StudioApp.js

```diff
@@ -356,7 +356,7 @@ StudioApp.prototype.init = function(config) {
   // Fixes viewport for small screens.
   var viewport = document.querySelector('meta[name="viewport"]');
   if (viewport) {
-    this.fixViewportForSmallScreens_(viewport, config);
+    this.fixViewportForSmallScreens_(viewport, 600);
```

Comment:
Is this intentional?  I don't think we changed this function from taking a config object...

Reply:
Good catch. Not intentional.